webapp/src/services/web3utils.js

```diff
@@ -0,0 +1,42 @@
+import { human_standard_token_abi } from "./human_standard_token_abi";
+
+const promisify = (inner) =>
+  new Promise((resolve, reject) =>
+    inner((err, res) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(res);
+      }
+    })
+  );
+
```

Comment:
I think we won't need this function,
we can use await directly in getBalance

Reply:
Removing that and getBalance since I don't think we need ETH balance for now